tools/deploy-datadog

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash -e
+
+cd ${JUPYTERHUB_DIR}/datadog
```

Comment:
this line isn't working for me while running `tools/deploy-datadog` from jupyterhub-deploy/